meltano.yml

```diff
@@ -0,0 +1,29 @@
+version: 1
+send_anonymous_usage_stats: true
+project_id: target-parquet
+default_environment: test
+environments:
+- name: test
+plugins:
+  extractors:
+  - name: tap-carbon-intensity
+    variant: meltano
+    pip_url: git+https://gitlab.com/meltano/tap-carbon-intensity.git
```

Comment:
What this extractor is about? 
Should we freeze its version?

Reply:
This is just an extract example that doesn't need any authentication. So the user can just tap `meltano install` and `meltano el tap-carbon-intensity target-parquet` to manually test the target (it's in the README example by default).
I don't think we need to freeze because the target should work regardless of the version.